docker/Dockerfile

```diff
@@ -6,13 +6,32 @@ RUN apk add --update git && \
     apk add curl && \
     apk add bash && \
     apk add openjdk8 && \
-    apk add openjdk8-doc
+    apk add openjdk8-doc && \
+    apk add expect
 
 # Set sbt and java env vars
 ENV SBT_VERSION 0.13.11
 ENV SBT_HOME /usr/local/sbt
 ENV PATH ${PATH}:${SBT_HOME}/bin
 ENV PATH ${PATH}:/usr/lib/jvm/java-1.8-openjdk/bin
+ENV SCALA_VERSION=2.11.8
+ENV SCALA_HOME=/usr/share/scala
+
+
+# Install Scala    credit: https://hub.docker.com/r/brs999/alpine-scala/~/dockerfile/
+RUN apk add --no-cache --virtual=.build-dependencies wget ca-certificates && \
+    apk add --no-cache bash && \
+    apk add --update alpine-sdk && \
```

Comment:
I doubt if `apk add --update alpine-sdk` is required. I'll try and update. I'll remove the duplicate `bash` stuff.